prod/westeurope/internal/api/functions_services_r3/function_app_slot_staging/terragrunt.hcl

```diff
@@ -99,6 +99,8 @@ inputs = {
     FETCH_KEEPALIVE_TIMEOUT             = "60000"
 
     SLOT_TASK_HUBNAME = "StagingTaskHub"
+
+    IO_FUNCTIONS_ADMIN_BASE_URL = http://api-internal.io.italia.it/
```

Comment:
ditto

Reply:
Fixed in [f7e24da](https://github.com/pagopa/io-infrastructure-live-new/pull/254/commits/f7e24da2d1f91b59a86295b9b4440e65df005c70)